### DISCUSSION TITLE
Progress is not calculating properly in self learning portal

### DISCUSSION THREAD

**Discussion by ghost** [github/india#367] [Students]
I have completed the first module[AZ-900] in self-learning. But still, it is not showing completed. I have completed it in sequential order as per instructions. Please look into the problem.

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Comment by Mik-27** [comment]
Same Issue

**Comment by ghost** [comment]
Make sure that all the modules are completed in step by step manner
